src/file/file.service.ts

```diff
@@ -141,12 +157,13 @@ export class FileService {
 
   private static async createUserRootFolder(userID: string): Promise<IFile> {
     const folder: IFile = {
-      type: 'Folder',
+      type: FolderContentType,
       key: null,
       bucket: userID,
       fullName: userID,
       ownerID: userID,
       isRootFolder: true,
+      deleted: false,
     };
     return await FilesRepository.create(folder);
   }
```

Comment:
Missing new line before the function's comment and after the end of the previous function.

Reply:
Same in line 173.